src/sql/src/plan/query.rs

```diff
@@ -1536,7 +1536,7 @@ fn plan_set_expr(
                 ShowStatement::ShowObjects(stmt) => {
                     show::show_objects(qcx.scx, stmt)?.plan_hir(qcx)
                 }
-                ShowStatement::ShowVariable(_) => sql_bail!("unsupported SHOW statement"),
+                ShowStatement::ShowVariable(_) => bail_unsupported!("SHOW variable"),
```

Comment:
Could we specify the context in which it's not supported, i.e. subqueries?

Reply:
Done.